.circleci/config.yml

```diff
@@ -31,4 +47,4 @@ workflows:
       - build
       - test:
           requires:
-            - build
+            - build
```

Comment:
Request for change - I think you use the convention of a newline at the end of the file?